app/ui.js

```diff
@@ -1744,7 +1744,7 @@ var UI = {
 };
 
 // Set up translations
-var LINGUAS = ["de", "el", "nl", "sv"];
+var LINGUAS = ["de", "el", "nl", "sv", "pl"];
```

Comment:
You forgot to update this list po/Makefile as well.

Also, could you place "pl" after "nl" but before "sv"? Just so that we can keep the list alphabetically ordered.